kornia/utils/structures.py

```diff
@@ -0,0 +1,53 @@
+from dataclasses import dataclass, field
```

Comment:
since it's a specific structure of sold2, rename it to `kornia/feature/sold2/structures.py`

Reply:
Oh, sorry, just saw that i missunderstood your initial comment! I move it!